sorbet/config

```diff
@@ -0,0 +1,2 @@
+rbi/
+--suppress-error-code=5002,5020,5035,5067
```

Comment:
We can remove this suppress now right https://github.com/Shopify/rbi-central/blob/533bdae78a3c1e6a5b669b9a77e1a3ebca7e4606/scripts/check_types#L19

Reply:
Actually you made me realize that we should be ignoring the config all together to avoid checking all the files in the same run.

I added `--no-config` to the script call.

Thanks for being so thorough Kaan ❤️